WordPress/Classes/WordPress.xcdatamodeld/.xccurrentversion

```diff
@@ -3,6 +3,6 @@
 <plist version="1.0">
 <dict>
 	<key>_XCCurrentVersionName</key>
-	<string>WordPress 79.xcdatamodel</string>
+	<string>WordPress 80.xcdatamodel</string>
```

Comment:
Nobody told you about it before, so it's by no means your fault — but it's customary to add a entry in `MIGRATIONS.md` every time we create a new Core Data model version. Would you be so kind and and one for this :)?

Reply:
Oh yes, now you said it I remember i did it once during the trial! Done! 😀